Taskfile.yml

```diff
@@ -178,21 +178,28 @@ tasks:
   test:acceptance:
     desc: "Run all acceptance tests (~90m)"
     deps:
-    - task: charts:kind-cluster
+    - task: test:pull-images
     - task: build:image
       vars:
         CLI_ARGS: '' # Don't forward CLI args to build:image
     vars:
       RUN: '{{ default `"^TestAcceptance"` .RUN }}'
       GO_TEST_RUNNER: '{{default "go test" .GO_TEST_RUNNER}}'
     cmds:
-    - kind load docker-image localhost/redpanda-operator:dev
     - task: test:unit
       vars:
         GO_TEST_RUNNER:
           ref: .GO_TEST_RUNNER
         CLI_ARGS: '{{.CLI_ARGS}} -run {{.RUN}} -timeout 35m -tags acceptance'
 
+  test:pull-images:
+    cmds:
+      - docker pull quay.io/jetstack/cert-manager-controller:v1.14.2
```

Comment:
Food for though: this is going to get out of sync really fast. Think it'd be easy to move these versions into envars in the flake.nix and refer to those across the codebase?